docs/simulations/jvb_hpa.py

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/python3
+"""
+This script is meant to model the behavior of the JVB nodepool on the cluster
+under the influence of the HPA.
+You may modify the constant values of the problem in the first section of the
```

Comment:
Instead of modifying, is it possible to override them using an environment variable ?